Automattic/woocommerce-payments
# Account Status block for an Account Overview page

Fixes #1409 

#### Changes proposed in this Pull Request

Implement an account status block with email, deposits and payments status and account fees.

Depends on #1375 to provide full currency name.

#### Testing instructions





 * Enable option `_wcpay_feature_account_overview`
 * Open `wp-admin` and go under **Payments** -> **Overview**
 * Ensure that rendered component matches proposed design: ITsL0PJ0Xz4ZzQpsWyMgiI-fi-1566%3A0

![Example](https://user-images.githubusercontent.com/683297/116705315-af09a700-a9cc-11eb-92be-cb931198d857.png)


> **Note:** I didn't move tests for `AccountFees` utility method yet, as they are more thoroughly tested with `AccountFees` component. However it should be done either in separate PR or within #1412.

-------------------
